_includes/navbar.html

```diff
@@ -59,7 +59,7 @@
                         <li><a href="http://www.apache.org/licenses/">License</a></li>
                         <li><a href="http://www.apache.org/foundation/contributing.html">Donations</a></li>
                         <li><a href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a></li>
-                        <li><a href="http://www.apache.org/security/">Report a Vulnerability</a></li>
+                        <li><a href="http://www.apache.org/security/">Security</a></li>
```

Comment:
@nacx Unrelated change - is this an update to the recommended links?

Reply:
Yes, it is unrelated but trivial so I just included it to match the navigation link policy: https://www.apache.org/foundation/marks/pmcs#navigation